data_collection/web/guides.rst

```diff
@@ -9,4 +9,224 @@ Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+.. note::
+  If you want to use CSS classes or HTML attributes to mark links as download or outlink and you have modified default JS snippet, make sure that :ref:`enableLinkTracking<tracking-outlink>` is called. It is enabled in default snippet, but if you use a custom one, then you have to enable it by yourself.
+
+```
+// Enable Download & Outlink tracking
+_paq.push(['enableLinkTracking']);
+```
+
+Downloads
+^^^^^^^^^
+
+Usually, you want to track when visitors downloads files from your website, a document, a file, a photo. Piwik PRO will automatically track clicks on such links as Downloads, and reports them in `Downloads` report.
+
+Our JS tracker is able to recognize when a click on a link is a download link.
+
+It will automatically recognize such when a clicked link contains one of following file extensions (extensions starts with "." character and one of following characters sets):
+
+```
+7z, aac, apk, arc, arj, asf, asx, avi, azw3, bin, bz, bz2, csv, deb, dmg, doc, docx, epub, exe, flv, gif, gz, gzip, hqx, ibooks, jar, jpg, jpeg, js, mp2, mp3, mp4, mpg, mpeg, mobi, mov, movie, msi, msp, odb, odf, odg, ods, odt, ogg, ogv, pdf, phps, png, ppt, pptx, qt, qtm, ra, ram, rar, rpm, sea, sit, tar, tbz, tbz2, tgz, torrent, txt, wav, wma, wmv, wpd, xls, xlsx, xml, z, zip
+```
+
+In one of following link schemas:
+
+ - file extension is at the very end of a link eg. `http://example.com/file.7z` or `http://example.com/article?click=file.7z`
+ - file extension directly preceeds query part (`?`), eg. `http://example.com/article/file.7z?source=user#how-to`
+ - file extension directly preceeds fragment part (`#`) `http://example.com/article?target=file.7z#how-to`
+ - file extension is at the end of query param, eg. `http://example.com/article?click=file.7z&page=3`
+
+Customizing list of file extensions
+"""""""""""""""""""""""""""""""""""
+You can customize list of file extansions you want to track as downloads. For example, if you want to track only images as downloads, you can use `setDownloadExtensions` function to replace the list this:
+
+```
+// track clicks on images only
+_paq.push(['setDownloadExtensions', "png|jpg|webp|gif"]);
+```
+
+You can add new extensions, to an existing list with `addDownloadExtensions`:
+
+```
+// add other image formats
+_paq.push(['setDownloadExtensions', "svg|xcf"]);
+```
+
+Or remove some of extenstions from the existing list with `removeDownloadExtensions`:
+
+```
+_paq.push(['removeDownloadExtensions', "jpg|jpeg"]);
+_paq.push(['trackPageView']);
```

Comment:
IS track page view nescessary here?

Reply:
Not, but it is like this in Matomo documentation. If you insist I can remove those extra `trackPageView`

Reply:
I will not fight about it